components/Post.vue

```diff
@@ -4,9 +4,13 @@
       deleted: post.is_deleted,
       'h-entry': detail
     }" class="media w-100 justify-content-start">
-      <nuxt-link :to="`/@${mainPost.user.username}`" v-if="!preview" v-bind:class="{ 'p-author h-card': detail }">
+      <a v-if="detail" v-show=false rel="author" :href="(mainPost.user.verified ? mainPost.user.verified.link : `/@${mainPost.user.username}`)" class="p-author h-card">
```

Comment:
Please use `<nuxt-link>` instead of `<a>` :bow:

Reply:
When the author of a post has a `user.verified` link, I want the link to point to that verified link (external). `<nuxt-link>` points at a link relative to the site, not an external link, right?

Reply:
Ah, I see.